mod/proxy.php

```diff
@@ -140,11 +140,23 @@ function proxy_init(App $a) {
 	if (!$direct_cache AND ($cachefile == '')) {
 		$r = qu("SELECT * FROM `photo` WHERE `resource-id` = '%s' LIMIT 1", $urlhash);
 		if (dbm::is_result($r)) {
+<<<<<<< HEAD
```

Comment:
Must be a committed half-merge. :-( Fixed it now by taking code from upstream/develop.